### DISCUSSION TITLE
What should I do when download servers are down?

### DISCUSSION THREAD

**Discussion by FaserGer853** [AloneLiberty/FlipperNested#24] [Q&A] [answered]
No, seriously. I can't build apps, I need to download the app for my Flipper Zero. Can someone help me?

**Comment by DeNcHiK3713** [comment, marked as answer]
[all-the-apps-base.zip](https://github.com/AloneLiberty/FlipperNested/files/13999621/all-the-apps-base.zip)
Here is the latest working version. Should work with unleashed firmware v 065, or official firmware v 0.93.0. For newer firmware versions, you need to wait for the code update as the official firmware code has changed and the application is not compatible with the new version.

**Reply by FaserGer853** [reply]
Thanks.
